File: Java/Gate-Eight/ContactApp.java
import java.util.*;

public class ContactApp {

    static void addContact() {
        System.out.print("Enter Name: ");
        String name = input.nextLine();

        System.out.print("Enter Phone: ");
        String phone = input.nextLine();

        names.add(name);
        phones.add(phone);

        System.out.println("Added!");
    }
  
    static void removeContact() {
        System.out.print("Enter phone Number to delete: ");
        String phone = input.nextLine();

        int index = phones.indexOf(phone);

        if (index != -1) {
            names.remove(index);
            phones.remove(index);
            System.out.println("Removed!");
        } else {
            System.out.println("Not found!");
        }
    }

    static void findContact() {
        System.out.print("Enter phone Number: ");
        String phone = input.nextLine();

        int index = phones.indexOf(phone);

        if (index != -1) {
            System.out.println("Name: " + names.get(index));
            System.out.println("Phone: " + phones.get(index));
        } else {
            System.out.println("Not found!");
        }
    }
}

File: Python/Gate-Eight/contact_app.py
contact_list = []

def add_contact(first_name, last_name,phone_Number):
    first_name =int(input("Enter First Name: ")
    last_name = int(input("Enter LasT Name: ")
    phone_number = int(input("Enter Phone Number: ")

    first_name.append(first_name)
    last_name.append(last_name)
    phone_number.append(phone_number)
    print "Contact Added Successfully"

def remove_contact(phone_number):
    for contact in contacts:
        if phone_number[contact] == contact_list:
            contact_list.remove(phone_number)
    print "Contact Removed Successfully"



